packages/mermaid/src/rendering-util/createText.ts

```diff
@@ -168,6 +168,21 @@ function updateTextContentAndStyles(tspan: any, wrappedLine: MarkdownWord[]) {
   });
 }
 
+/**
+ *
+ * @param text - The raw string to adjust
+ * @returns
+ */
+
+// Used for converting substrings in node labels/edges/text into fontawesome icons by using a regex pattern
+// The letters 'bklrs' stand for possible endings of the fontawesome prefix (e.g. 'fab' for brands, 'fak' for fa-kit) // cspell: disable-line
+export function replaceIconSubstring(text: string) {
```

Comment:
Minor fix, to use proper jsdoc comments. 

```suggestion
/**
 * Convert fontawesome labels into fontawesome icons by using a regex pattern
 * @param text - The raw string to convert
 * @returns string with fontawesome icons as i tags
 */
export function replaceIconSubstring(text: string) {
  // The letters 'bklrs' stand for possible endings of the fontawesome prefix (e.g. 'fab' for brands, 'fak' for fa-kit) // cspell: disable-line
```